exercise3-1/src/camera_demo/include/utils.h

```diff
@@ -0,0 +1,14 @@
+#ifndef ROSCON_2023_REALTIME_WORKSHOP_UTILS_H_
+#define ROSCON_2023_REALTIME_WORKSHOP_UTILS_H_
+#include <cactus_rt/utils.h>
+
+#include <chrono>
+
+// A funtion to imply take up some time, pretending some processing has occured.
+inline void WasteTime(std::chrono::microseconds duration) {
+  const auto start = cactus_rt::NowNs();
+  auto       duration_ns = duration.count() * 1000;
+  while (cactus_rt::NowNs() - start < duration_ns) {
+  }
```

Comment:
I am fine with this. Works well. 

You could also add a function call, like [examples_rclcpp_cbg_executor/pong_node.cpp#L75](https://github.com/ros2/examples/blob/1d97c4fc7445554f6f85f63305d424fc017212a0/rclcpp/executors/cbg_executor/src/examples_rclcpp_cbg_executor/pong_node.cpp#L75) to do really some computing.